src/com/dotmarketing/business/web/WebContext.java

```diff
@@ -0,0 +1,48 @@
+package com.dotmarketing.business.web;
+
+import com.dotcms.AppContext;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+
+/**
+ * Created by freddyrodriguez on 31/8/16.
+ */
+public class WebContext implements AppContext {
```

Comment:
WebContext does not says so much about what is wrap on it.
I would prefer WebSessionContext, it will be easier to the user to determine which is the intention
we could have also a WebRequestContext, etc.